src/goto-synthesizer/cegis_verifier.cpp

```diff
@@ -89,6 +89,15 @@ void cegis_verifiert::preprocess_goto_model()
     goto_model, log.get_message_handler(), cprover_cpp_library_factory);
   link_to_library(
     goto_model, log.get_message_handler(), cprover_c_library_factory);
+  // library functions may introduce inline assembler
+  while(has_asm(goto_model))
```

Comment:
No need for regression tests here either?

Reply:
If we can get acceptable performance out of FreeBSD CI runs, we could exercise this. Still trying...